examples/siro_sandbox/VR_HITL.md

```diff
@@ -0,0 +1,204 @@
+# VR Human-in-the-loop (HITL) Evaluation
+
+HITL evaluation can be driven from VR. In this mode, the HITL app acts as a server that can be remotely accessed by a client.
+
+As it stands, the VR integration can only be used with the `pick_throw_vr` app state. In this mode, the user controls a human avatar that can interact with policy-driven Spot robot.
+
+This document explains how to set up and troubleshoot the VR integration.
+
+## Table of Contents
+  - [Installation](#installation)
+    - [Requirements](#requirements)
+      - [Server](#server)
+      - [Client](#client)
+  - [Launch Command](#launch-command)
+  - [Unity Data Folder](#unity-data-folder)
+    - [Dataset Processing](#dataset-processing)
+      - [Installation](#installation-1)
+      - [Usage](#usage)
+      - [Importing Data Into Unity](#importing-data-into-unity)
+  - [Running Locally from Unity](#running-locally-from-unity)
+  - [Running Remotely from Quest Headset](#running-remotely-from-quest-headset)
+    - [Connection](#connection)
+    - [Server Controls](#server-controls)
+  - [Troubleshooting](#troubleshooting)
+    - [Deployment to the VR Headset](#deployment-to-the-vr-headset)
+    - [Connection Issues](#connection-issues)
+    - [Slow Performance](#slow-performance)
+
+
+## Installation
+
+The system is composed of the following components:
+
+* The **Server**, which is the HITL app.
+* The **Client**, which runs on Unity and can be deployed to a VR headset.
+
+### Requirements
+
+#### Server
+
+| Requirements | Notes |
+|---|---|
```

Comment:
We should just link to `https://github.com/facebookresearch/habitat-lab/blob/hitl-vr/examples/siro_sandbox/README.md#installation` for how to install the HITL app. That should cover installing habitat-lab and habitat-sim dependencies and most of the required data.